README.md

```diff
@@ -0,0 +1,124 @@
+    and incrementing by 1. For example, if the highest numbered PR is 125, your
+    RFC number would be 126. You should end up with a folder name like
+    `0026-my-cool-feature`.
+  - Create the folder and copy `0000-template.md` to `text/<your folder name>/README.md`.
+  - Fill in the RFC. Put care into the details: RFCs that do not present
+    convincing motivation, demonstrate an understanding of the impact of the
+    design, or are disingenuous about the drawbacks or alternatives tend to be
+    poorly received. You can add supporting artifacts, such as diagrams and sample
+    data, in the RFC's folder.
+  - Submit a pull request.
+
+Make sure that all of your commits satisfy the [DCO requirements](
+https://github.com/probot/dco#how-it-works) of the repo.
+
+The RFC Maintainers will check to see if the process has been followed, and request any process changes before merging the PR. 
+
+When the PR is merged, your RFC is now in the PROPOSED state.
+
+### How to get an RFC accepted
+
+After your RFC has been proposed, the RFC will receive feedback from the larger
+community, and the author should be prepared to revise it. Updates may be made via pull request, and those changes will be merged as long as the process is followed.
+
+When you believe that the RFC is mature enough (feedback is somewhat resolved,
+consensus is emerging, and implementation against it makes sense), submit a PR that changes the Status to ACCEPTED. The status change PR will remain open until the maintainers agree on the status change.
+
+>NOTE: contributors who used the Indy HIPE process prior to May 2019 should
+see the acceptance process substantially simplified under this approach.
+The bar for acceptance is not perfect consensus and all issues resolved;
+it's just general agreement that a doc is "close enough" that it makes
+sense to put it on a standards track where it can be improved as
+implementation teaches us what to tweak.
+
+### How to get an RFC adopted
+
+An accepted RFC is a standards-track document. It becomes an acknowledged
+standard when there is evidence that the community is deriving meaningful
+value from it. So:
+
+- Implement the ideas, and find out who else is implementing.
+- Socialize the ideas. Use them in other RFCs and documentation.
+- Update the agent test suite to reflect the ideas.
+
+When you believe an RFC is a _de facto_ standard, raise a PR that changes the status to ADOPTED. 
+the __adopted__ status. If the community is friendly to the idea, the doc
+will enter a two-week "Final Comment Period" (FCP), after which there will
+be a vote on disposition.
+
+## About
+
+#### License
+
+This repository is licensed under an [Apache 2 License](LICENSE).
+
+#### Contributions
+
+Unless you explicitly state otherwise, any contribution intentionally submitted
+for inclusion in the repo by you, as defined in the Apache-2.0 license, shall be
+dual licensed as above, without any additional terms or conditions.
```

Comment:
I see the Apache License above. What do you mean by dual licensed?

Reply:
typo from source document. fixed. thanks!